tests/eas/acceptance.py

```diff
@@ -133,7 +133,7 @@ def _do_test_first_cpu(self, experiment, tasks):
                 rank=len(tasks)),
             msg="Not all the new generated tasks started on a big CPU")
 
-class ForkMigration(unittest.TestCase):
+class ForkMigration(EasTest):
```

Comment:
As @JaviMerino used to say: I :heart_eyes: these massively red coloured patches ;-)